.ci.yaml

```diff
@@ -949,7 +949,7 @@ targets:
       add_recipes_cq: "true"
       dependencies: >-
         [
-          {"dependency": "android_sdk", "version": "version:33v6"},
+          {"dependency": "android_sdk", "version": "version:UDCv1"},
```

Comment:
For the sake of future us can you add a comment to one of these files that UDCv1 is upsideDownCake and link to the cipd dependency directly?

Reply:
Added a comment explaining